src/content/vagrant/install-landing.json

```diff
@@ -10,6 +10,7 @@
 			"os": "darwin"
 		}
 	],
+	"sidebarMenuItems": [],
```

Comment:
Prevents error in `product-downloads-view` caused by `sidebarMenuItems` being undefined